calendar-redis/src/main/java/org/apache/james/events/CalendarChangeEvent.java

```diff
@@ -0,0 +1,41 @@
+/********************************************************************
+ *  As a subpart of Twake Mail, this file is edited by Linagora.    *
+ *                                                                  *
+ *  https://twake-mail.com/                                         *
+ *  https://linagora.com                                            *
+ *                                                                  *
+ *  This file is subject to The Affero Gnu Public License           *
+ *  version 3.                                                      *
+ *                                                                  *
+ *  https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ *                                                                  *
+ *  This program is distributed in the hope that it will be         *
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ *  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ *  PURPOSE. See the GNU Affero General Public License for          *
+ *  more details.                                                   *
+ ********************************************************************/
+
+package org.apache.james.events;
+
+import org.apache.james.core.Username;
+
+public record CalendarChangeEvent(EventId eventId) implements Event {
```

Comment:
Is this event serializable? (DTO converter stuff?)

BTW do not put it in calendar-redis. storage-api?